components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -69,6 +69,7 @@ public class IdentityRecoveryConstants {
     public static final String RECOVERY_STATUS_INCOMPLETE = "INCOMPLETE";
     public static final String RECOVERY_STATUS_COMPLETE = "COMPLETE";
     public static final String TEMPLATE_TYPE = "TEMPLATE_TYPE";
+    public static final String CUSTOM_TEMPLATE_NAME = "CUSTOM_TEMPLATE_NAME";
```

Comment:
let's not use `custom`

```suggestion
    public static final String EMAIL_TEMPLATE_NAME = "templateName";
```

Reply:
Fixed with d4678fff1d2d5b0c7f780d8179ecf1769f72c057